TypeSystem/flow-analysis/reachability_A07_t08.dart

```diff
@@ -2,29 +2,23 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion If N is an expression of the form E1.m1(E2), then:
+/// @assertion Method invocation: If `N` is an expression of the form
+/// `E1.m1(E2)`, then:
+///  - Let `before(E1) = before(N)`
+///  - Let `before(E2) = after(E1)`
+///  - Let `T` be the static return type of the invocation
+///  - If `T <: Never` then:
+///   - Let `after(N) = unreachable(after(E2))`.
+///  - Otherwise:
+///    - Let `after(N) = after(E2)`
 ///
-///  Let before(E1) = before(N)
-///  Let before(E2) = after(E2)
-///  Let T be the static return type of the invocation
-///  If T <: Never then:
-///    Let null(N) = unreachable(before(N)).
-///    Let notNull(N) = unreachable(before(N)).
-///  Otherwise if T <: Null then:
-///    Let null(N) = before(N).
-///    Let notNull(N) = unreachable(before(N)).
-///  Otherwise if T is non-nullable then:
-///    Let null(N) = before(N).
-///    Let notNull(N) = unreachable(before(N)).
-///  Otherwise:
-///    Let null(N) = promote(x, Null, before(N))
-///    Let notNull(N) = promoteToNonNull(x, before(N))
-///
-/// @description Checks reachability after method call. Test static method and
-/// non-nullable return type
+/// @description Checks reachability after method invocation. Tests a static
+/// method with a non-nullable return type.
 /// @author sgrekhov@unipro.ru
 /// @issue 41985
 
+// @dart = 3.8
```

Comment:
Again, a comment here reminding the reader that we need `@dart = 3.8` in order to ensure that there is no sound-flow-analysis.

Reply:
Same in a couple of other libraries.